### DISCUSSION TITLE
Space character in secret.system value

### DISCUSSION THREAD

**Discussion by nurzhan86** [ory/hydra#2606] [Q&A] [answered]
We use hydra 1.8.5 in k8s cluster, and we want to upgrade it to latest 1.10.2 version.
It so happened that the current system.secret value contains a string with space characters, for example: "abc 12345 678 9 def 987654".
And hydra 1.8.5 works great with that, no errors during or after install.

But when we trying to upgrade hydra to version 1.10.2, the error:

`The configuration contains values or keys which are invalid:
secrets: map[cookie:[[abc 12345 678 9 def 98765]] system:[[abc 12345 678 9 def 98765]]]
         ^-- validation failed

The configuration contains values or keys which are invalid:
secrets.system: [[abc 12345 678 9 def 98765]]
                ^-- validation failed

The configuration contains values or keys which are invalid:
secrets.system.0: [abc
                  ^-- length must be >= 16, but got 4

The configuration contains values or keys which are invalid:
secrets.system.1: 12345
                  ^-- length must be >= 16, but got 5

The configuration contains values or keys which are invalid:
secrets.system.2: 678
                  ^-- length must be >= 16, but got 3
...
and so on... `

Hydra 1.10.2 think that my current system.secret value is a multi-value secret and there are all less 16 length, but in version 1.8.5 it is a single value and work fine.

How to tell to hydra 1.10.2 to work with space character contained value in system.secret?
How to upgrade hydra to version 1.10.2 with current system.secret value containing space characters?

**Comment by zepatrik** [comment, marked as answer]
Locking as this bug is tracked as #2609
